source/aws_signing.c

```diff
@@ -476,59 +530,59 @@ static int s_add_authorization_query_params(struct aws_signing_state_aws *state,
         return AWS_OP_SUCCESS;
     }
 
+    int result = AWS_OP_ERR;
+
+    struct aws_byte_buf uri_encoded_value;
+    AWS_ZERO_STRUCT(uri_encoded_value);
+    if (aws_byte_buf_init(&uri_encoded_value, state->allocator, 256)) {
```

Comment:
where's 256 come from, should it be a constant?

Reply:
Yes